docs/references/protocol/ledger-data/ledger-entry-types/oracle.md

```diff
@@ -0,0 +1,90 @@
+# Oracle
+
+_(Requires the [PriceOracle amendment][] {% not-enabled /%})_
+
+[[Source]](https://github.com/XRPLF/rippled/blob/master/src/ripple/protocol/impl/LedgerFormats.cpp#L353-L366 "Source")
+
+An `Oracle` ledger entry holds data associated with a single price oracle object.
+
+{% admonition type="info" name="Note" %}
+
+A price oracle object can store information for up to 10 token pairs.
+
+{% /admonition %}
+
+
+## Example Oracle JSON
+
+```json
+{
+  "LedgerEntryType": "Oracle",
+  "Owner": "rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW",
+  "Provider": "70726F7669646572",
+  "AssetClass": "63757272656E6379",
+  "PriceDataSeries": [
+    {
+      "PriceData": {
+        "BaseAsset": "XRP",
+        "QuoteAsset": "USD",
+        "AssetPrice": 74,
+        "Scale": 2,
+      }
+    },
+  ],
+  "LastUpdateTime": 743609414,
+  "PreviousTxnID": "C53ECF838647FA5A4C780377025FEC7999AB4182590510CA461444B207AB74A9",
+  "PreviousTxnLgrSeq": 56865244
+}
+```
+
+
+## Oracle Fields
+
+| Field               | JSON Type | Internal Type | Required? | Description |
+|---------------------|-----------|---------------|-----------|-------------|
+| `Owner`             | String    | AccountID     | Yes       | The XRPL account with update and delete privileges for the oracle. It's recommended to set up [multi-signing](https://xrpl.org/set-up-multi-signing.html) on this account. |
+| `Provider`          | String    | Blob          | Yes       | An arbitrary value that identifies an oracle provider, such as Chainlink, Band, or DIA. This field is a string, up to 256 ASCII hex encoded characters (0x20-0x7E). |
+| `PriceDataSeries`   | Array     | Array         | Yes       | An array of up to 10 `PriceData` objects, each representing the price information for a token pair. More than five `PriceData` objects require two owner reserves. |
+| `LastUpdateTime`    | Number    | UInt32        | Yes       | The time the data was last updated, represented in Unix time. |
```

Comment:
As I also noted in `get_aggregate_price`, the example doesn't make sense as Unix time; it would represent a "last-updated" date of July 25, 1993.

If it's intended to be seconds since the Ripple Epoch, it would be a little closer (July 25, 2023) but still a bit implausible.